setupAMGXMFEM.sh

```diff
@@ -0,0 +1,18 @@
+#cd ../ &&
+#wget https://computing.llnl.gov/projects/hypre-scalable-linear-solvers-multigrid-methods/download/hypre-2.10.0b.tar.gz &&
+#tar -zxvf hypre-2.10.0b.tar.gz &&
+#mv hypre-2.10.0b hypre &&
+#cd hypre/src/ && mkdir build && cd build &&
+#cmake -DCMAKE_INSTALL_PREFIX=$(pwd)/../../ ../ && make -j && make install &&
+
+cd ../ &&
```

Comment:
This file is pretty helpful but probably doesn't belong in the repo. Maybe a very brief instruction on installing AMGx could be put in the changelog or readme?

Reply:
deal!

Reply:
This should probably go in `INSTALL`